src/ontology/mondo-ingest.Makefile

```diff
@@ -326,7 +326,7 @@ documentation: j2 $(ALL_DOCS) unmapped-terms-docs mapped-deprecated-terms-docs s
 
 .PHONY: build-mondo-ingest
 build-mondo-ingest:
-	$(MAKE) refresh-imports exclusions-all mondo-ingest.db slurp-all mappings matches \
+	$(MAKE) refresh-imports exclusions-all merged.db slurp-all mappings matches \
```

Comment:
## `merged.db` within `build-mondo-ingest`
@hrshdhgd Nico doesn't know; do you know why `merged.db` (previously called `mondo-ingest.db` even though it comes from `tmp/merged.owl`) is part of `build-mondo-ingest`?

@matentzn  @hrshdhgd I'm guessing maybe this was added because we just want to mark it as some required output / something that is needed for release / analysis. But I personally think this should be removed from `build-mondo-ingest`; it stands out as the only artefact that is declared in that goal, and I just don't see any reason for it.

Reply:
I think we had that there during a meeting when Nico was dictating recipes to me. I don't recall me being a decision maker for this one.

Reply:
If you are positive that there are no _hidden_ dependencies on this file (thinking, python scripts, use grep to find out), then I am fine removing it for now.

Reply:
Sounds good. Based on what you and Harshad are saying, and my investigations, and us not finding anything about it in the release during our meeting, it looks like it was put there by mistake, so I will delete it.